### DISCUSSION TITLE
Part clarification

### DISCUSSION THREAD

**Discussion by ccfman2004** [sanni/cartreader#465] [Need help]
For the 1K and 220 Ohm resistors, is there a specific set of values I need to get like power rating and resistor type? For the LED, are there specific values I need to look for like power rating? What kind of size restraints are there for components?

Sorry for the questions but I'm used to being given a specific BOM that tells me the exact values of the parts I need.

**Comment by sanni** [comment]
You can use standard metal film with 1/4W like these https://www.aliexpress.com/item/1005002670881002.html
For the LED any 3mm LED in the color you like, max 20mA https://www.aliexpress.com/item/1005003664666225.html

**Reply by ccfman2004** [reply]
Thank you and I see you updated the "What to Order" page to clarify what I was asking.
